client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -173,7 +172,8 @@ const DataFilesToolbar = ({ scheme, api }) => {
   const canRename = getFilePermissions('rename', permissionParams);
   const canMove = getFilePermissions('move', permissionParams);
   const canCopy = getFilePermissions('copy', permissionParams);
-  const canTrash = getFilePermissions('trash', permissionParams);
+  const inTrash = path.includes('.Trash');
+  const canTrash = getFilePermissions('trash', permissionParams) && !inTrash;
```

Comment:
May need to use `startsWith()` instead of `includes()` because there could be edge cases in which the `path` might have `.Trash` (e.g. `/example.Trash/exampleFile/`). In this case, `exampleFile` would not be deleteable.

It would also be nice if it could be in one line to keep the `can*` list consistent.